src/Posts.jsx

```diff
@@ -1,54 +1,49 @@
 const GRAPHQL_ENDPOINT =
   props.GRAPHQL_ENDPOINT || "https://near-queryapi.api.pagoda.co";
-const LIMIT = 25;
-let accountsFollowing = props.accountsFollowing;
+const LIMIT = 10;
```

Comment:
Let's decrease `LIMIT` from 25 to 10? I don't see any reason why do we need to load 25 posts in fact that on 1 page could possibly fit something about 10 or even 8 posts.

Reply:
@shelegdmitriy please move this UX change and the "There are not any posts yet" change into separate PRs so we can get feedback from other people on those changes without holding up the fix in this PR.

Reply:
Sure I can revert `LIMIT = 25` but If I'll remove "There are not any posts yet" placeholder then **"Following"** tab will be empty. If I'll revert filtering logic (I mean we'll show posts from "all" tab on "following") then this PR doesn't make any sence. How about compromise? Let's add a line under "There are not any posts yet" with some text which will point to https://near.org/near/widget/PeoplePage ?

cc @charleslavon

Reply:
Like you proposed here https://github.com/near/near-discovery-components/pull/369#discussion_r1342728857

Reply:
Good call @shelegdmitriy. We can iterate from there with feedback from @TiffanyGYJ

Reply:
Instead of "There are not any posts yet" please use "Build your feed by finding [people to follow](https://near.org/near/widget/PeoplePage)"